Board Shape

adamjvr opened the pull request:
Added board shape and mocked up location of Kimchi module and lipo battery solution.

adamjvr committed:
Board Shape

Added board shape and mocked up location of Kimchi module and lipo battery solution.

mwelling commented:
Looks good. You might want to move it up a bit to be flush with the edge of the board to make the USB port more easily accessible.

IE
![image](https://user-images.githubusercontent.com/1570196/101437347-d9938000-38c4-11eb-9620-b666e139fd81.png)

adamjvr committed:
Mockup Progress

Added standard SMT tactile switches to represent the button pad layout for both D-Pad and Face Buttons. Added dummy Analog Sticks for mockup purposes until we can get samples of the Nintendo Switch Joy Con replacement sticks. Also added USB port and Audio Codec/Headset jack, currently working through codec shecmatic implemenation.
